apps/docs/content/guides/auth/auth-mfa/webauthn.mdx

```diff
@@ -0,0 +1,370 @@
+
+There are two main types of authenticators:
+
+1. **Platform Authenticators**: Built into the user's device
+   - Examples:
+     - Desktop: Touch ID (macOS), Windows Hello (Windows)
+     - Mobile: Face ID (iOS), Biometrics (Android)
+2. **Roaming Authenticators**: Portable devices that work across multiple platforms
+   - Examples: YubiKey, Google Titan Security Key, Solokeys
+
+## Authentication Flows
+
+WebAuthn involves two primary processes: Enrollment and Login. Both follow a similar pattern:
+
+<Image
+  alt="Diagram showing the flow of Multi-Factor authentication"
+  src={{
+    light: '/docs/img/guides/auth-mfa/auth-mfa-webauthn-flow.svg',
+    dark: '/docs/img/guides/auth-mfa/auth-mfa-webauthn-flow.svg',
+  }}
+  containerClassName="max-w-[700px]"
+/>
+
+### Step-by-Step Explanation
+
+1. The user's device initiates the process by requesting a challenge from the server.
+2. The server generates and sends a unique challenge to the authenticator.
+3. The authenticator prompts the user for verification.
+4. The user provides the required verification (e.g., fingerprint, face scan).
+5. Upon successful user verification, the authenticator signs the challenge using its private key.
+6. The server validates the signed challenge using the corresponding public key.
+7. If the validation is successful, the authentication process is completed.
+
+This flow applies to both enrollment (when setting up WebAuthn) and login (when using WebAuthn to authenticate).
+
+**Navigator API**
+
+In both flows, WebAuthn interacts with the browser through the Navigator API, specifically using the `navigator.credentials` interface. This API provides methods like `create()` for registering new credentials and `get()` for authenticating existing ones. By handling the secure communication with authenticators (such as hardware tokens or biometrics), the Navigator API simplifies the implementation of WebAuthn, allowing developers to focus on integrating strong, passwordless authentication into their web applications.
+
+### Add enrollment flow
+
+An enrollment flow provides a UI for users to set up additional authentication factors. Most applications add the enrollment flow in two places within their app:
+
+1. Right after login or sign up.
+   This allows users quickly set up Multi Factor Authentication (MFA) post login or account creation. Where possible, encourage all users to set up MFA. Many applications offer this as an opt-in step in an
+   effort to reduce on-boarding friction.
+2. From within a settings page.
+   Allows users to set up, disable or modify their MFA settings.
+
+As far as possible, maintain a generic flow that you can reuse in both cases with minor modifications.
+
+You can enroll a WebAuthn factor with a single step:
+
+```ts
+import { browserSupportsWebAuthn } from '@supabase/supabase-js'
```

Comment:
Does this need to be imported? Looks like it is a method on `supabase.auth` from the example below?

Reply:
Thanks! You're right the method `browserSupportsWebAuthn` shouldn't have a `supabase.auth` prefix